pages/apply/students.js

```diff
@@ -34,6 +34,8 @@ class Students extends Component {
           buttonLink={data.applicationLink}
           minorButtonLink="https://www.notion.so/h4iuiuc/Join-Hack4Impact-UIUC-2e875ce396b34e6ebb23c6dda57a89aa"
           minorButtonText="View Positions"
+          isLeftButtonDisplayed={false}
+          isRightButtonDisplayed={true}
         />
         <ApplicationProcess
```

Comment:
pass in `closed` prop here (just add a line after the `processes` prop and type `closed`.